samples/java-spring-cinema-booking-choreography/src/test/java/com/example/cinema/DomainGenerators.java

```diff
@@ -0,0 +1,43 @@
+package com.example.cinema;
+
+import com.example.cinema.model.Show;
+
+import java.math.BigDecimal;
+import java.util.Random;
+import java.util.UUID;
+
+import static com.example.cinema.ShowBuilder.showBuilder;
+
+
+public class DomainGenerators {
+
+  private static final Random random = new Random();
+
+  public static String randomShowId() {
+    return UUID.randomUUID().toString();
+  }
+
+  public static String randomWalletId() {
+    return UUID.randomUUID().toString();
+  }
+
+  public static String randomReservationId() {
+    return UUID.randomUUID().toString();
+  }
```

Comment:
Kind of odd to have three methods abstracting away random uuid and not adding anything new.

Reply:
This is only logica abstraction but can be removed